### ISSUE TITLE
mintty osc 52 paste over ssh no response

### ISSUE THREAD

**Issue by theimpostor** [mintty/mintty#1301]
In a mintty console on the local machine, if I print the osc 52 paste terminal sequence I see the expected response echoed to the terminal:

```bash
# In Windows 11 mintty session
# Clipboard contents are 'hello world'

$ echo -e "\e]52;;?\e\\"

$ 52;;aGVsbG8gd29ybGQ=

# response is correct
$ echo aGVsbG8gd29ybGQ= | base64 -d
hello world
```

However if I ssh into a linux machine I get no response written to the console:

```bash
# Same mintty console, ssh into linux

❯ echo -e "\e]52;;?\e\\"

# no response
❯
```
 Using mintty 3.7.6, cygwin 3.5.4, Windows 11. `TERM=mintty` in both environments. .minttyrc contents:

```
Term=mintty
AllowSetSelection=true
AllowPasteSelection=true
StatusLine=yes
BellType=0
```

How can I debug this further?

**Comment by mintty** [comment]
Works here. Are you sure you put something into the _Windows_ clipboard? If you copy to some X11 clipboard on the remote side, that's not what mintty would respond.

**Comment by theimpostor** [comment]
Yes the windows clipboard is has the string and the remote linux host is a
headless VM.

Does the ssh app make a difference, i.e. should it work if using the built
in windows ssh.exe instead of cygwin openssh? I used windows ssh.exe in my
test.

**Comment by mintty** [comment]
That might explain it. You shouldn't use Windows tools in a terminal! I haven't tried but it likely blocks certain escape sequences (same with wsl.exe, that's why wslbridge exists).

**Comment by theimpostor** [comment]
Looks like ssh.exe was the culprit - I tried with cygwin openssh and paste worked. Thanks!
